### ISSUE TITLE
crashes on startup and several files are failing to be read

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/open-ig#14]
```
What steps will reproduce the problem?
1.just start the game
2.
3.

What is the expected output? What do you see instead?
it crashes half the time on startup, just see the loading screen constantly. if 
i kill java.exe in the processes then sometimes it starts to work otherwise i 
just have to keep trying till it eventually works

What version of the product are you using? On what operating system?
0.91

Please provide any additional information below.

```

Original issue reported on code.google.com by `puddyc...@gmail.com` on 21 Jun 2011 at 1:08

**Comment by GoogleCodeExporter** [comment]
```
java.util.zip.ZipException: error in opening zip file
    at java.util.zip.ZipFile.open(Native Method)
    at java.util.zip.ZipFile.<init>(Unknown Source)
    at java.util.zip.ZipFile.<init>(Unknown Source)
    at hu.openig.core.ResourceLocator.analyzeZip(ResourceLocator.java:185)
    at hu.openig.core.ResourceLocator.scanResources(ResourceLocator.java:165)
    at hu.openig.core.Configuration.newResourceLocator(Configuration.java:370)
    at hu.openig.screen.CommonResources.init(CommonResources.java:181)
    at hu.openig.screen.CommonResources.<init>(CommonResources.java:177)
    at hu.openig.GameWindow.<init>(GameWindow.java:265)
    at hu.openig.Startup$4.run(Startup.java:163)
    at java.awt.event.InvocationEvent.dispatch(Unknown Source)
    at java.awt.EventQueue.dispatchEventImpl(Unknown Source)
    at java.awt.EventQueue.access$000(Unknown Source)
    at java.awt.EventQueue$1.run(Unknown Source)
    at java.awt.EventQueue$1.run(Unknown Source)
    at java.security.AccessController.doPrivileged(Native Method)
    at java.security.AccessControlContext$1.doIntersectionPrivilege(Unknown Source)
    at java.awt.EventQueue.dispatchEvent(Unknown Source)
    at java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source)
    at java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source)
    at java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source)
    at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
    at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
    at java.awt.EventDispatchThread.run(Unknown Source)


This the code i get when i load up after killing the javaw.exe process and the 
game is playable
```

Original comment by `puddyc...@gmail.com` on 21 Jun 2011 at 1:09

**Comment by GoogleCodeExporter** [comment]
```
Did you use the launcher to download and install the application?

Have you tried a more ancient version version of the game? If so, you may 
delete the open-ig-config.xml to get rid of possibly bad settings.

Do you start it with the proper memory amount?
java -Xmx512M -jar open-ig-0.91.jar
```

Original comment by `akarn...@gmail.com` on 21 Jun 2011 at 7:31
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Hello,

I did use the launcher to download IG, I did try the orignal game from
packaging but had uninstalled and reboot afterwards before downloading this
version. I have the memory settings all running nice and dandy.. so not sure
where to go from here.

is there anything i can send that you might want to look at?
```

Original comment by `puddyc...@gmail.com` on 26 Jun 2011 at 8:31

**Comment by GoogleCodeExporter** [comment]
```
right here is the first set of code that come up

java.util.zip.ZipException: error in opening zip file
    at java.util.zip.ZipFile.open(Native Method)
    at java.util.zip.ZipFile.<init>(Unknown Source)
    at java.util.zip.ZipFile.<init>(Unknown Source)
    at hu.openig.core.ResourceLocator.analyzeZip(ResourceLocator.java:185)
    at
hu.openig.core.ResourceLocator.scanResources(ResourceLocator.java:165)
    at
hu.openig.core.Configuration.newResourceLocator(Configuration.java:370)
    at hu.openig.screen.CommonResources.init(CommonResources.java:181)
    at hu.openig.screen.CommonResources.<init>(CommonResources.java:177)
    at hu.openig.GameWindow.<init>(GameWindow.java:265)
    at hu.openig.Startup$4.run(Startup.java:163)
    at java.awt.event.InvocationEvent.dispatch(Unknown Source)
    at java.awt.EventQueue.dispatchEventImpl(Unknown Source)
    at java.awt.EventQueue.access$000(Unknown Source)
    at java.awt.EventQueue$1.run(Unknown Source)
    at java.awt.EventQueue$1.run(Unknown Source)
    at java.security.AccessController.doPrivileged(Native Method)
    at java.security.AccessControlContext$1.doIntersectionPrivilege(Unknown
Source)
    at java.awt.EventQueue.dispatchEvent(Unknown Source)
    at java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source)
    at java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source)
    at java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source)
    at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
    at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
    at java.awt.EventDispatchThread.run(Unknown Source)


To get this next set of code i clicked.. EN > Single player > New game (left
it on normal)

java.util.zip.ZipException: error in opening zip file
    at java.util.zip.ZipFile.open(Native Method)
    at java.util.zip.ZipFile.<init>(Unknown Source)
    at java.util.zip.ZipFile.<init>(Unknown Source)
    at hu.openig.core.ResourceLocator.analyzeZip(ResourceLocator.java:185)
    at
hu.openig.core.ResourceLocator.scanResources(ResourceLocator.java:165)
    at
hu.openig.core.Configuration.newResourceLocator(Configuration.java:370)
    at hu.openig.screen.CommonResources.init(CommonResources.java:181)
    at hu.openig.screen.CommonResources.reinit(CommonResources.java:291)
    at hu.openig.GameWindow.switchLanguage(GameWindow.java:436)
    at hu.openig.screen.items.MainScreen$11.act(MainScreen.java:366)
    at
hu.openig.screen.items.MainScreen$ClickLabel.invoke(MainScreen.java:76)
    at hu.openig.screen.items.MainScreen.mouse(MainScreen.java:216)
    at hu.openig.GameWindow$MouseActions.invoke(GameWindow.java:1159)
    at hu.openig.GameWindow$MouseActions.mouseReleased(GameWindow.java:1171)
    at java.awt.Component.processMouseEvent(Unknown Source)
    at javax.swing.JComponent.processMouseEvent(Unknown Source)
    at java.awt.Component.processEvent(Unknown Source)
    at java.awt.Container.processEvent(Unknown Source)
    at java.awt.Component.dispatchEventImpl(Unknown Source)
    at java.awt.Container.dispatchEventImpl(Unknown Source)
    at java.awt.Component.dispatchEvent(Unknown Source)
    at java.awt.LightweightDispatcher.retargetMouseEvent(Unknown Source)
    at java.awt.LightweightDispatcher.processMouseEvent(Unknown Source)
    at java.awt.LightweightDispatcher.dispatchEvent(Unknown Source)
    at java.awt.Container.dispatchEventImpl(Unknown Source)
    at java.awt.Window.dispatchEventImpl(Unknown Source)
    at java.awt.Component.dispatchEvent(Unknown Source)
    at java.awt.EventQueue.dispatchEventImpl(Unknown Source)
    at java.awt.EventQueue.access$000(Unknown Source)
    at java.awt.EventQueue$1.run(Unknown Source)
    at java.awt.EventQueue$1.run(Unknown Source)
    at java.security.AccessController.doPrivileged(Native Method)
    at java.security.AccessControlContext$1.doIntersectionPrivilege(Unknown
Source)
    at java.security.AccessControlContext$1.doIntersectionPrivilege(Unknown
Source)
    at java.awt.EventQueue$2.run(Unknown Source)
    at java.awt.EventQueue$2.run(Unknown Source)
    at java.security.AccessController.doPrivileged(Native Method)
    at java.security.AccessControlContext$1.doIntersectionPrivilege(Unknown
Source)
    at java.awt.EventQueue.dispatchEvent(Unknown Source)
    at java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source)
    at java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source)
    at java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source)
    at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
    at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
    at java.awt.EventDispatchThread.run(Unknown Source)
Exception in thread "Movie Video" java.lang.NullPointerException
    at hu.openig.screen.VideoRenderer.run(VideoRenderer.java:77)

I hope this helps
```

Original comment by `puddyc...@gmail.com` on 26 Jun 2011 at 8:35

**Comment by GoogleCodeExporter** [comment]
```
What is your operating system exactly?

Can you open the zip files manually?

If your operating system treats zip files as directories, it could be a 
problem. You should turn that feature off.
```

Original comment by `akarn...@gmail.com` on 26 Jun 2011 at 9:00

**Comment by GoogleCodeExporter** [comment]
```
My OS is windows Vista 32bit ultimate

I can open the zip files manually.

Not sure how to turn that feature off as i use winrar as my zip reader. any
help on that would be kinda helpful
```

Original comment by `puddyc...@gmail.com` on 26 Jun 2011 at 2:07

**Comment by GoogleCodeExporter** [comment]
```
I've uploaded a new version with possible workaround:

http://open-ig.googlecode.com/files/open-ig-0.92b.jar

You can run this version only outside the launcher for now:

java -Xmx512M -jar open-ig-0.92b.jar

It should solve your problem.



```

Original comment by `akarn...@gmail.com` on 26 Jun 2011 at 8:11

**Comment by GoogleCodeExporter** [comment]
```
Hopefully fixed in v0.92 downloadable via the launcher.
```

Original comment by `akarn...@gmail.com` on 19 Jul 2011 at 9:18
- Changed state: **Fixed**
